internal/core/platform.rs

```diff
@@ -282,6 +282,9 @@ pub fn duration_until_next_timer_update() -> Option<core::time::Duration> {
 pub use crate::input::key_codes::Key;
 pub use crate::input::PointerEventButton;
 
+// reexport focus reason enum to the public api
+pub use crate::input::FocusEventReason;
```

Comment:
I think it doesn't need to be in the public API.

The test can keep using the private API

Reply:
Sorry, still getting acquainted with the project structure, how would I access it via the private API in the test?

Reply:
The test can use `slint::private_unstable_api::re_exports::FocusEventReason`